catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -196,6 +196,9 @@ protected Object executeWithSubject() throws Exception {
 
                 entry = zipInputStream.getNextEntry();
             }
+        } catch (Exception e) {
```

Comment:
I don't believe so. The outer method is throwing Exception which means any exception that gets thrown bubbles out to it. This is meant to ensure we log the exception instead of trusting whatever the karaf command code does with it.